src/base/BaseActionsRouter.sol

```diff
@@ -27,13 +27,13 @@ abstract contract BaseActionsRouter is SafeCallback {
     /// @notice function that is called by the PoolManager through the SafeCallback.unlockCallback
     function _unlockCallback(bytes calldata data) internal override returns (bytes memory) {
         // abi.decode(data, (uint256[], bytes[]));
```

Comment:
update the comment

also can you add a @param tag explaining the encoding for actions